src/utils/workspace.ts

```diff
@@ -0,0 +1,27 @@
+import * as vscode from "vscode";
+
+interface IExtTextEditor extends vscode.TextEditor {
+    id: string;
+}
+
+export async function checkTextFileIsOpened(path: string) {
```

Comment:
Are we sure this does not interfere with the user work flow - like blinking a window or activating some other window than currently selected?

Reply:
It will be, as I showed you, but unfortunately there's no other way for us to achieve the requested functionality.